app/controller/sdl/RCModulesController.js

```diff
@@ -0,0 +1,926 @@
+
+    /**
+     * @name moduleIDMapping
+     * @type {Map}
+     * @description Mapping module ID with driver and front passenger seat
+     */
+    moduleIDMapping: {},
+
+    /**
+     * @name moduleUUIDMapping
+     * @type {Map}
+     * @description Mapping module ID to UUID
+     */
+    moduleUUIDMapping: {},
+
+    /**
+     * @description Mapping of user friendly seat names and their key names
+     * @type {Map}
+     */
+    seatKeyLabelMapping: {},
+
+    /**
+     * @description Reference to currently active seat model
+     * @type {Object}
+     */
+    currentSeatModel: null,
+    
+    /**
+     * @description Reference to currently active audio model
+     * @type {Object}
+     */
+    currentAudioModel: null,
+
+    /**
+     * @description Reference to currently active climate model
+     * @type {Object}
+     */
+    currentClimateModel: null,
+
+    /**
+     * @description Reference to currently active radio model
+     * @type {Object}
+     */
+    currentRadioModel: null,
+
+    /**
+     * @description Reference to currently active hmi settings model
+     * @type {Object}
+     */
+    currentHMISettingsModel: null,
+
+    /**
+     * @description Reference to currently active light model
+     * @type {Object}
+     */
+    currentLightModel: null,
+
+    /**
+     * @description Function for controller initialization
+     */
+    init: function() {
+        // Mock models required for early binding initialization
+        this.set('currentAudioModel', SDL.AudioModel.create());
+        this.set('currentClimateModel', SDL.ClimateControlModel.create());
+        this.set('currentSeatModel', SDL.SeatModel.create({ID: 'No emulation mode. ID is undefined'}));
+        this.set('currentRadioModel', SDL.RadioModel.create());
+        this.set('currentHMISettingsModel', SDL.HmiSettingsModel.create());
+        this.set('currentLightModel', SDL.LightModel.create());
+    },
+
+    /**
+     * @function resetToDefault
+     * @description Reset all parameters of current object to default
+     */
+    resetToDefault: function() {
+      SDL.remoteControlCapabilities = SDL.deepCopy(SDL.initialRemoteControlCapabilities);
+      this.set('audioModels', {});
+      this.set('climateModels', {});
+      this.set('radioModels', {});
+      this.set('seatModels', {});
+      this.set('hmiSettingsModels', {});
+      this.set('lightModels', {});
+      this.set('moduleModelsMapping', {});
+      this.set('moduleIDMapping', {});
+      this.set('moduleUUIDMapping', {});
+      this.set('seatKeyLabelMapping', {});
+      this.init();
+    },
+    /**
+     * @description Function for generating a coverage according to specified
+     * settings and saving into the moduleModelsMapping  
+     * @param {String} module_type
+     * @param {Array} module_coverage 
+     */
+    fillModuleModelsMapping: function(module_type, module_coverage) {
+      var mapping = {};
+
+      module_coverage.forEach(module => {
```

Comment:
Since the elements in this array are the serviceArea of each module of the specified `module_type`
```suggestion
      module_coverage.forEach(module_service_area => {
```

Reply:
@ShobhitAd agree, it's more descriptive in this context. Fixed in https://github.com/smartdevicelink/sdl_hmi/pull/216/commits/ebb28ee4a9126c070b33a161e56a0fc09f82c516